packages/driver/src/cy/commands/xhr.js

```diff
@@ -471,6 +471,13 @@ module.exports = (Commands, Cypress, cy, state, config) => {
           })
         }
 
+        // look ahead to see if fixture exists
+        const fixturesRe = /^(fx:|fixture:)/
+
+        if (hasResponse && fixturesRe.test(options.response)) {
+          return cy.fixture(options.response.replace(fixturesRe, '')).then(() => route())
```

Comment:
please split regex replacement into its own line for clarity
```suggestion
          const fixtureName = options.response.replace(fixturesRe, '')
          return cy.fixture(fixtureName).then(() => route())
```

Reply:
Should be `cy.now('fixture', fixtureName).then` or else it's queueing the command, not running it.

Reply:
Done, thanks :)